pages/api/userCars/[id].js

```diff
@@ -0,0 +1,27 @@
+//even needed now?
```

Comment:
do you need this file?

Reply:
Deleted it already last time, somehow it has found his way back 😅

Reply:
sneaky file, if it happens again, reach out to us :)

Reply:
Thanks, i will if it keeps coming back